dev-util/hip/hip-5.1.3-r1.ebuild

```diff
@@ -4,8 +4,10 @@
 EAPI=8
 
 PYTHON_COMPAT=( python3_{8..11} )
+DOCS_BUILDER="doxygen"
+DOCS_DEPEND="media-gfx/graphviz"
 
-inherit cmake llvm prefix python-any-r1
+inherit cmake docs llvm prefix python-any-r1
 
 LLVM_MAX_SLOT=14
```

Comment:
What's this?

Reply:
I followed the example of `llvm.eclass`, which recognize `LLVM_MAX_SLOT`.

Reply:
Now the only usable llvm/clang SLOT is 14 for ROCm toolchain. In the future, we can unset this variable, and specify DEPEND as `>=sys-devel/clang-14.0.6-r1`